package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "contentful-ui-extensions-sdk",
-  "version": "3.25.0",
+  "version": "3.25.1",
```

Comment:
Should this be a minor version? I guess it technically is adding new functionality